completion_grading/edxapp_wrapper/completion.py

```diff
@@ -0,0 +1,17 @@
+"""
+Completion Service module generalized definitions.
+"""
+
+from importlib import import_module
+
+from django.conf import settings
+
+
+def init_completion_service(*args, **kwargs):
+    """
+    Wrapper for `submissions.api.create_submission` function in edx-submissions.
+    """
+    backend_function = settings.COMPLETION_SERVICE_BACKEND
```

Comment:
Shouldn't the format be `COMPLETION_GRADING_<backend_name>_BACKEND`?

Reply:
I changed the name :). Thanks!